src/components/Release/Release.css

```diff
@@ -0,0 +1,38 @@
+.Release {
+  display: flex;
+  justify-content: center;  
+  align-items: center;
+  flex-direction: column;
+  border-radius: 10px;
+  background-color: #ffb367;
+  color: white;
+  text-align: center;
+  font-family: Georgia, 'Times New Roman', Times, serif;
```

Comment:
I've seen this font-family a few times, consider putting this in a more global scope (an index.css perhaps) and then using {font-family: "inherit"} if needed for "a" tags and others elements that don't automatically inherit fonts